app/assets/stylesheets/store.scss

```diff
@@ -5,10 +5,12 @@
     list-style: none;
```

Comment:
Make sure there are no extra changes in a PR

Reply:
Okay, will take care of it